common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2207,6 +2207,8 @@ public static enum ConfVars {
         "Whether to use codec pool in ORC. Disable if there are bugs with codec reuse."),
     HIVE_USE_STATS_FROM("hive.use.stats.from","iceberg","Use stats from iceberg table snapshot for query " +
         "planning. This has three values metastore, puffin and iceberg"),
+    HIVE_COL_STATS_SOURCE("hive.col.stats.source","metastore","Use stats from puffin file for  query " +
```

Comment:
- do we support `iceberg` mode? 
- please update the description - "Use stats from selected source for query planning"

Reply:
Fixed, merged the confs to a single conf.